src/test/java/com/iexec/core/result/ResultServiceTests.java

```diff
@@ -63,10 +70,10 @@ class ResultServiceTests {
     private Credentials schedulerCreds;
     private Signature signature;
     private WorkerpoolAuthorization workerpoolAuthorization;
+    private TaskDescription taskDescription;
```

Comment:
instead of recreating the object for each test, you could do:
```suggestion
    private final TaskDescription taskDescription = TaskDescription.builder()
        ...
        .build();
```